### PR TITLE
Removed support for Cantabular version 9.2.0

### PR DESCRIPTION
**Pull request by phynes-sensiblecode**
`9.2.0` is now an old version of Cantabular- the scripts only had support for this version to aid with early development.

This PR removes support for generating files in version `9.2.0` format in order to reduce the maintenance overhead.

### CODE REVIEW

File: bin/ons_csv_to_ctb_json_main.py [hunk 1]
```diff
@@ -185,18 +184,7 @@ def main():
...
-                         FILE_CONTENT_TYPE_TABLES]
-
-    if args.cantabular_version == DEFAULT_CANTABULAR_VERSION:
-        logging.info(
-            f'Output files will be written in Cantabular {args.cantabular_version} format')
-
-    elif args.cantabular_version == CANTABULAR_V9_2_0:
-        output_file_types = [FILE_CONTENT_TYPE_DATASET, FILE_CONTENT_TYPE_SERVICE]
-        convert_json_to_ctb_v9_2_0(ctb_datasets, ctb_tables, service_metadata)
+    if args.cantabular_version in KNOWN_CANTABULAR_VERSIONS:
```
Comment (mikejamesthompson) [inline bin/ons_csv_to_ctb_json_main.py:187]: Maybe clarify that even if we write our their version string verbatim (e.g. 10.1.0), this is actually the same as 10.2.2 format.

File: README.md [hunk 2]
```diff
@@ -331,12 +331,12 @@ Other Cantabular versions
 =========================
 
 The `-v` argument can be used to generate output files that are compatible with a different version of Cantabular.
-At present only 9.2.0, 9.3.0, 10.0.0, 10.1.0, 10.1.1, 10.2.0, 10.2.1 and 10.2.2 are supported. If any other version is specified then the specified version
+At present only 9.3.0, 10.0.0, 10.1.0, 10.1.1, 10.2.0, 10.2.1 and 10.2.2 are supported. If any other version is specified then the specified version
 will be reflected in the output filenames, but `10.2.2` format will be used.
 
-To generate version 9.2.0 compatible files from the test data use the following command:
+To generate JSON files from the test data with version `99.9.9` reflected in the filenames use the following command:
```
[the remark this answers is not in the document]
Comment (mikejamesthompson) [reply README.md:337]: Maybe test the behaviour instead?

[the remark this answers is not in the document]
Comment (mikejamesthompson) [reply README.md:337]: (Not essential though!)

**Review by mikejamesthompson — APPROVED** [review]
Looks good, couple of small comments

File: bin/ons_csv_to_ctb_json_main.py (hunk 1 above)
Comment (phynes-sensiblecode) [reply bin/ons_csv_to_ctb_json_main.py:187]: I'll add a log like this:
```
t=2023-01-13 15:44:44,281 lvl=INFO msg=Output files will be written in Cantabular 10.2.2 format, which is compatible with all versions of Cantabular from 9.3.0 to 10.2.2
```

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (phynes-sensiblecode) [reply README.md]: I've removed the example from `README.md`. I haven't added a test.

**Review by mikejamesthompson — APPROVED** [review]
👍

**Pull request merged**